app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -27,6 +27,7 @@ class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
 
     var mLocationManager: LocationManager? = null
     var mLocationListener: LocationManager? = null
+    private val PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION: Int = 0
```

Comment:
Created constant variable declaration as once value is created it should not be reassigned